pkg/bootkube/bootstrap.go

```diff
@@ -0,0 +1,84 @@
+package bootkube
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/kubernetes-incubator/bootkube/pkg/asset"
+)
+
+// CreateBootstrapControlPlane seeds static manifests to the kubelet to launch the bootstrap control
+// plane.
+func CreateBootstrapControlPlane(assetDir string, podManifestPath string) error {
+	UserOutput("Running temporary bootstrap control plane...\n")
+
+	// Make secrets temporarily available to bootstrap cluster.
+	if err := os.RemoveAll(asset.BootstrapSecretsDir); err != nil {
+		return err
+	}
+	if err := os.Mkdir(asset.BootstrapSecretsDir, os.FileMode(0755)); err != nil {
+		return err
+	}
+	secretsDir := filepath.Join(assetDir, asset.AssetPathSecrets)
+	secrets, err := ioutil.ReadDir(secretsDir)
+	if err != nil {
+		return err
+	}
+	for _, secret := range secrets {
+		if err := copyFile(filepath.Join(secretsDir, secret.Name()), filepath.Join(asset.BootstrapSecretsDir, secret.Name()), true); err != nil {
+			return err
+		}
+	}
+
+	// Copy the static manifests to the kubelet's pod manifest path.
+	manifestsDir := filepath.Join(assetDir, asset.AssetPathBootstrapManifests)
```

Comment:
I might start with leaving the restart policy as default (always). If we don't need to provide coordination points (e.g. x starts before y) it keeps bootkube somewhat more simple -- and the api-server should just keep retrying until it can access etcd. This is a common pattern across kubernetes.